src/misc/ibvwrap.cc

```diff
@@ -140,64 +140,71 @@ ncclResult_t wrap_ibv_symbols(void) {
   return initResult;
 }
 
-#define IBV_PTR_CHECK_ERRNO(name_internal, call, retval, error_retval, name) \
+#define IBV_PTR_CHECK_ERRNO_NO_RETURN(name_internal, call, retval, error_retval, name, res) \
```

Comment:
Actually if we move ALL calls to those "no return" macros then maybe we don't need to rename them, just change their behavior?

So I'd even remove the _NO_RETURN everywhere.

Reply:
Makes sense. I modified the patch to remove _NO_RETURN. Can you please take a look? Thank you.